### ISSUE TITLE
Can I learn the embedding of individual nodes? 

### ISSUE THREAD

**Issue by chang861224** [facebookresearch/SEAL_OGB#23]
Hi @muhanzhang,

I am a master student at NCCU. I saw your SEAL method on OGB link property prediction task, and I am interested in this method. But I have a trivial question, hope you can solve my doubt. 

I have read your code and I  know that your SEAL method learn the embedding for subgraphs constructed from each pair of nodes. However, I wanna know that, can this method learn the embedding of individual nodes rather than the embedding of edges? Or it can only be used for learning a pair of nodes (i.e. edges)?

**Comment by muhanzhang** [comment]
No. Using node labeling methods and enclosing subgraphs can also help node/graph representation learning other than edge learning. See this [paper](https://arxiv.org/pdf/2012.01380.pdf) and this [one](https://arxiv.org/pdf/2110.13197.pdf).
